core/Http.php

```diff
@@ -96,8 +96,13 @@ public static function sendHttpRequest($aUrl,
         // create output file
         $file = self::ensureDestinationDirectoryExists($destinationPath);
 
+        $acceptInvalidSslCertificate = false;
+        if (isset(Config::getInstance()->Development['accept_invalid_ssl'])) {
```

Comment:
@mikkeschiren could you add a default value to the `config/global.ini.php` for `accept_invalid_ssl`? And also add `Development::isEnabled()` to the check?

Reply:
@mikkeschiren any chance you could check out my comment above and add the default value?